test/test_bundle.js

```diff
@@ -841,6 +1035,53 @@ describe('json-pollock tests', function () {
 
   });
 
+  describe('render carousel select', function () {
+    const container = addToBody(JsonPollock.render(JSON.stringify(carouselSelectConf)));
+    const carouselRoot = container.children[0];
+    const carouselRootWrapper = container.children[0].children[0];
+    const carouselRootLayout = container.children[0].children[0];
+    const carouselListRoot =  carouselRootLayout.children[0];
+    const card1 = carouselRootLayout.children[0].children[0];
+    const card2 = carouselRootLayout.children[0].children[1];
+    const card3 = carouselRootLayout.children[0].children[2];
+
+    it('carousel root exist', function () {
+      chai.expect(carouselRoot.className).to.contain('lp-json-pollock');
+    });
+
+    it('carousel wrapper root exist', function () {
+      chai.expect(carouselRootWrapper.className).to.contain('lp-json-pollock-layout-carousel-wrapper');
+    });
+
+    it('carousel root layout exist', function () {
+      chai.expect(carouselRootLayout.className).to.contain('lp-json-pollock-layout-carousel');
+    });
+
+    it('carousel has aria-label', function(){
+      chai.expect(carouselListRoot.getAttribute('aria-label')).to.be.equal('Carousel with buttons');
+    });
```

Comment:
@dvakulenko Hey, this test seems to be not relevant anymore as I don't see this label gets added anywhere in the logic